HerPublicWebsite.BusinessLogic/ExternalServices/EmailSending/GovUkNotifyApi.cs

```diff
@@ -135,8 +132,34 @@ string followUpLink
             
 
         }
-    }
+        
+        public void SendComplianceEmail(
+                MemoryStream recentReferralRequestOverviewFileData,
+                MemoryStream recentReferralRequestFollowUpFileData,
+                MemoryStream historicReferralRequestFollowUpFileData
+                )
+                {
+                    var template = govUkNotifyConfig.ComplianceReportTemplate;
+                    Dictionary<String, dynamic> personalisation = new Dictionary<String, dynamic>
+                    {
+                        { "File1Link", NotificationClient.PrepareUpload(recentReferralRequestOverviewFileData.ToArray(), true)},
+                        { "File2Link", NotificationClient.PrepareUpload(recentReferralRequestFollowUpFileData.ToArray(), true)},
+                        { "File3Link", NotificationClient.PrepareUpload(historicReferralRequestFollowUpFileData.ToArray(), true)},
+                    };
+                    foreach (var emailAddress in govUkNotifyConfig.ComplianceEmailRecipients)
+                    {
+                        var emailModel = new GovUkNotifyEmailModel
+                        {
+                            EmailAddress = emailAddress,
+                            TemplateId = template.Id,
+                            Personalisation = personalisation
+                        };
+                        SendEmail(emailModel);
```

Comment:
Does Gov UK Notify allow you to send to multiple recipients? Can we stick them all on the to 'To' line, or do we explicitly want to send separate emails for each one?

Reply:
Looks like it doesn't, so unfortunately we'll just have to loop through.